helm/squidex/values.yaml

```diff
@@ -32,29 +32,71 @@ ingress:
   #      - chart-example.local
 
 env:
-  URLS__BASEURL: null
+  URLS__BASEURL: null # https://squidex.local/
   URLS__ENFORCEHTTPS: false
   EVENTSTORE__CONSUME: true
   EVENTSTORE__TYPE: MongoDb
-  IDENTITY__ADMINEMAIL: null
-  IDENTITY__ADMINPASSWORD: null
-  IDENTITY__SHOWPII: true
   LETSENCRYPT_HOST: null
   LETSENCRYPT_EMAIL: null
-  LOGGING__HUMAN: "false"
+  LOGGING__LEVEL: INFORMATION #Trace, Debug, Information, Warning, Error, Fatal
+  LOGGING__HUMAN: true #Setting the flag to true, enables well formatteds json logs.
+  LOGGING__APPLICATIONINSIGHTS__ENABLED: false
```

Comment:
Either add all telemetry settings/providers or none.

Reply:
added logging settings to values file.